docs/docs/getting-started/Installation.md

```diff
@@ -0,0 +1,24 @@
+---
+id: Installation
+sidebar_position: 1
+---
+
+This section will guide you through the process of getting started with Piscina.js. We'll cover installation, basic usage, and configuration options to help you quickly integrate Piscina.js into your project.
+
+To get started, you'll need to have Node.js version 16.x or higher installed on your system. Open your terminal and run the following command:
+
+```bash tab={"label":"npm"}
+npm install piscina
+```
+```bash tab={"label":"yarn"}
+yarn add piscina
+```
+```bash tab={"label":"pnpm"}
+pnpm add piscina
+```
+```bash tab={"label":"Bun"}
+bun add piscina
+```
```

Comment:
We just need to be careful about `bun` and say that for installing its good, but using within `bun` is not assured.

Reply:
Oh alright, I will include that.